html/janus.js

```diff
@@ -2182,7 +2182,11 @@ function Janus(gatewayCallbacks) {
 						// The new experimental getDisplayMedia API is available, let's use that
 						// https://groups.google.com/forum/#!topic/discuss-webrtc/Uf0SrR4uxzk
 						// https://webrtchacks.com/chrome-screensharing-getdisplaymedia/
-						navigator.mediaDevices.getDisplayMedia({ video: true, audio: media.captureDesktopAudio })
+						constraints.video = { frameRate: media.screenshareFrameRate };
```

Comment:
We should change the default `media.screenshareFrameRate = 3;` we have a few lines above, or maybe get rid of it entirely, before this can be merged: at the moment, since we ignore that property the video is reasonably fluid, but enforcing that default 3fps will make the experience quite crappy.

Reply:
I'd say let's just get rid of it, and let the browser do whatever it wants by default.